### DISCUSSION TITLE
자바스크립트 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by sdr03095** [changdoc/naver-vaccine-macro#253] [Q&A]
자바스크립트 링크를 수정하여 시도하였습니다.
링크를 넣었을 때 nn개 리스트에서 잔여백신 예약시도중 화면이 뜨는데, 위에 자동 예약 시작 버튼을 누르면 다시 병원 1개 대상으로 하는 크롬 창으로 넘어갑니다ㅠㅠ
'nn개 리스트에서 잔여백신 예약시도중' 창에서 위에 자동 예약 신청을 누르지 않아도 되는 건지 궁금합니다.

**Comment by changdoc** [comment]
https://github.com/kimytsc/covid-rest-vaccine-macro <-- 이 프로젝트에 가셔서 문의 해보셔야할 것 같습니다.
